.circleci/config.yml

```diff
@@ -33,6 +33,12 @@ jobs:
             - .tox/py*
             - /root/.cache/pip/wheels/
 
+      - run:
+          name: Black style check
+          command: |
+            pip3 install --user black
```

Comment:
can't we do this as part of the requirements.txt?

Reply:
I added it first in the requirements, but the black binary is for some reason not recognized by the CI server. Also directly calling it from `~/.local/bin/` or with `python3 -m black` didn't work when I put it into the `requirements.txt` and the `setup.py` file.

Maybe it's also better this way since black's dependencies are more cleanly separated from what you actually need to run Mythril. What do you think?

Reply:
Oay that's fine